torch/_torch_docs.py

```diff
@@ -7637,13 +7655,17 @@ def merge_dicts(*dicts):
 
     >>> torch.remainder(torch.tensor([-3., -2, -1, 1, 2, 3]), 2)
     tensor([ 1.,  0.,  1.,  1.,  0.,  1.])
-    >>> torch.remainder(torch.tensor([1, 2, 3, 4, 5]), 1.5)
-    tensor([ 1.0000,  0.5000,  0.0000,  1.0000,  0.5000])
+    >>> torch.remainder(torch.tensor([1, 2, 3, 4, 5]), -1.5)
+    tensor([ -0.5000, -1.0000,  0.0000, -0.5000, -1.0000 ])
```

Comment:
Adding a negative sign for denominator helps show the difference between `fmod` and `remainder`. For `remainder`, the output's sign is same as denominator unlike `fmod` (same sign as dividend or numerator).